### PR TITLE
[Tizen] Fix compile error

### PR DESCRIPTION
**Pull request by pyrasis**
- Fix compile error.

### CODE REVIEW

File: extensions/LocalStorage/LocalStorage.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 #include "cocos2d.h"
 
-#if (CC_TARGET_PLATFORM != CC_PLATFORM_ANDROID)
+#if (CC_TARGET_PLATFORM != CC_PLATFORM_ANDROID && CC_TARGET_PLATFORM != CC_PLATFORM_TIZEN)
```
Comment (dumganhar) [inline extensions/LocalStorage/LocalStorage.cpp:32]: Tizen port has its own implementation of localStorage?

Comment (pyrasis) [inline extensions/LocalStorage/LocalStorage.cpp:32]: It is temporarily.
I will prepare and add sqlite3 library for Tizen x86 and armv7l.

Comment (dumganhar) [inline extensions/LocalStorage/LocalStorage.cpp:32]: OK

**Comment by dumganhar** [comment]
Merged. Thanks.

**Pull request merged**